test/e2e/spec.js

```diff
@@ -5,6 +5,6 @@ describe('First time claim flow', () => {
     return browser.url('/')
 
       // Index
-      .waitForExist('#temporary')
+      .waitForExist('#new-claims')
```

Comment:
we should make a seed data script to add test data for this e2e test to allow viewing claim info